admin-manual/maintenance/maintenance.rst

```diff
@@ -134,72 +142,287 @@ The index names are: `aips`, `aipfiles`, `transfers` and `transferfiles`.
+
+  service elasticsearch restart
+
+To use a new directory as snapshot repository, create and adjust permissions for one, like so:
+
+.. code:: bash
+
+  mkdir /var/lib/elasticsearch/backup-repo/es_backup_clientname
+  chmod 0755 /var/lib/elasticsearch/backup-repo/es_backup_clientname
+  chown elasticsearch:elasticsearch /var/lib/elasticsearch/backup-repo/es_backup_clientname
+
+Before any snapshot or restore operation can be performed, a snapshot repository
+should be registered in Elasticsearch. The repository settings are
+repository-type specific:
+
+.. code:: bash
+
+  curl -XPUT 'http://localhost:9200/_snapshot/es_backup_clientname' -d '{
+      "type": "fs",
+      "settings": {
+          "compress" : true,
+          "location": "/var/lib/elasticsearch/backup-repo/es_backup_clientname"
+      }
+  }'
+
+**Backing up Elasticsearch indexes**
+
+To make a backup (snapshot) for the ``aips`` and ``transfer`` indexes, a different name
+has to be used every time a snapshot is taken. For example, using the date
+inside the filename:
+
+.. code:: bash
+
+  curl -XPUT "http://localhost:9200/_snapshot/es_backup_clientname/snapshot_aips_20170726?pretty?wait_for_completion=true" -d'
+  {
+    "indices": "aips",  
+    "ignore_unavailable": true,
+    "include_global_state": false
+  }'
+
+  curl -XPUT "http://localhost:9200/_snapshot/es_backup_clientname/snapshot_transfers_20170726?pretty?wait_for_completion=true" -d'
+  {
+    "indices": "transfers",  
+    "ignore_unavailable": true,
+    "include_global_state": false
+  }'
+
+The snapshot will be saved to the
+``/var/lib/elasticsearch/backup-repo/es_backup_clientname`` directory. This
+directory can be backed up, for example, using rsync:
+
+.. code:: bash
+
+  rsync -av /var/lib/elasticsearch/backup-repo/es_backup_clientname /backup/location/elasticsearch
+
+To list all the snapshots:
+
+.. code:: bash
+
+  curl -XGET "https://localhost:9200/_snapshot/es_backup_clientname/_all"
+
+To delete a snapshot:
+
+.. code:: bash
+
+  curl DELETE "https://localhost:9200/_snapshot/es_backup_clientname/snapshot_transfers_20170726"
+
+**Restoring Elasticsearch**
+
+Before restoring, the snapshot repo has to be registered in elasticsearch (see
+preconfiguration). It can be restored in a different server, configuring the
+repo.path, registering the snapshot repo (different paths and repo names can be
+used) and copying the files inside the ``/backup/location/elasticsearch``
+directory.
+
+The index will have to be closed before restoration can occur. To close the
+index, post to the following _close endpoints, like so:
+
+.. code:: bash
+
+  curl -XPOST 'http://localhost:9200/aips/_close'
+  curl -XPOST 'http://localhost:9200/transfers/_close'
+
+To restore ElasticSearch:
+
+.. code:: bash
+
+  curl -XPOST 'http://localhost:9200/_snapshot/es_backup_clientname/snapshot_aips_20170726/_restore'
+  curl -XPOST 'http://localhost:9200/_snapshot/es_backup_clientname/snapshot_transfers_20170726/_restore'
```

Comment:
If the backup has been created with all the indexes, this could be done in one too:

```
curl -XPOST 'http://localhost:9200/_snapshot/es_backup_clientname/snapshot-am-YYYY.MM.DD/_restore'
```